setup.py

```diff
@@ -8,11 +8,11 @@
     "colorlog",
     "cookiecutter",
     "cryptography",
-    "docstring-parser",
+    "docstring-parser>=0.8",
```

Comment:
did we fix it already?

Reply:
https://github.com/inmanta/inmanta-core/pull/2984/files